src/App.tsx

```diff
@@ -34,42 +28,7 @@ function App(props: AppProps) {
       : undefined;
   });
 
-  const router = useRouter();
-
-  const routerReplace = useCallback((url: string) => {
```

Comment:
For those wondering, this has been moved to here: https://github.com/statelyai/xstate-viz/pull/245/files#diff-88a36bc7a53bfa58c6f451464798d631d785160dfacfdaf9479be7a0b1d7e5e5R34-R65